packages/pinia/test-dts/state.test-d.ts

```diff
@@ -67,6 +68,8 @@ expectType<number>(store.fromARef)
 
 expectType<{ msg: string }>(store.aShallowRef)
 expectType<{ msg: string }>(store.$state.aShallowRef)
+expectType<{ aRef: Ref<string> }>(store.anotherShallowRef)
+expectType<{ aRef: Ref<string> }>(store.$state.anotherShallowRef)
```

Comment:
This was passing even without my change because types are correctly set when using option stores